.github/workflows/listener.yml

```diff
@@ -12,34 +12,44 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      # - name: Echo Event
+      # - name: Echo Event >>>>>>>> something with these echos is broken for now
       #   run: | 
       #     echo "${{ toJson(github) }}"
       
-      - uses: ./.github/actions/replace_specs
-        id: replace-specs
+      - uses: ./.github/actions/replace_specs_snippets
+        id: replace-specs-snippets
         with: 
           branch-name: ${{ github.event.client_payload.branchName }}
           username: ${{ secrets.DX_GITHUB_USERNAME }}
           token: ${{ secrets.DX_GITHUB_TOKEN }}
       
-      - uses: ./.github/actions/add_pr_comment
+      - name: Comment on api-specs
+        uses: ./.github/actions/add_pr_comment
         with: 
           username: ${{ secrets.DX_GITHUB_USERNAME }}
           token: ${{ secrets.DX_GITHUB_TOKEN }}
           repo-name: api-specs
           pr-number: ${{ github.event.client_payload.prNumber }}
-          message: 'test'
+          message: 'https://bw-api-docs-${{ github.event.client_payload.branchName }}.s3-website-us-east-1.amazonaws.com/\n\nPlease note that it may take a couple minutes for your preview site to become available. See the corresponding PR opened on the docsite repository here:\nhttps://github.com/Bandwidth/api-docs/pull/${{ steps.replace-specs.outputs.pr-number }}'
+
+      - name: Comment on api-docs-code-snippets
+        uses: ./.github/actions/add_pr_comment
+        with: 
+          username: ${{ secrets.DX_GITHUB_USERNAME }}
+          token: ${{ secrets.DX_GITHUB_TOKEN }}
+          repo-name: api-docs-code-snippets
+          pr-number: ${{ github.event.client_payload.prNumber }}
+          message: 'https://bw-api-docs-${{ github.event.client_payload.branchName }}.s3-website-us-east-1.amazonaws.com/\n\nPlease note that it may take a couple minutes for your preview site to become available. See the corresponding PR opened on the docsite repository here:\nhttps://github.com/Bandwidth/api-docs/pull/${{ steps.replace-specs.outputs.pr-number }}'
```

Comment:
```suggestion
          message: 'https://bw-api-docs-${{ github.event.client_payload.branchName }}.s3-website-us-east-1.amazonaws.com/\n\nPlease note that it may take a couple minutes for your preview site to become available. See the corresponding PR opened on the docsite repository here:\nhttps://github.com/Bandwidth/api-docs/pull/${{ steps.replace-specs-snippets.outputs.pr-number }}'
```